sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+- Adds new get/set property maxMessageSizeInKilobytes to QueueProperties, TopicProperties, CreateQueueOptions, and CreateTopicOptions. Only applicable for premium namespaces. Added in [#17953](https://github.com/Azure/azure-sdk-for-js/pull/17953)
```

Comment:
do you support maxMessageSizeInKilobytes in update Queue/Topic options?

Reply:
yes

Reply:
```suggestion
- Adds new get/set property `maxMessageSizeInKilobytes` to `QueueProperties`, `TopicProperties`, `CreateQueueOptions`, and `CreateTopicOptions`. Only applicable for premium namespaces. Use this when creating or updating queues and topics. Added in [#17953](https://github.com/Azure/azure-sdk-for-js/pull/17953)
```